Sources/Core/Internal/TextureErrorHandler.swift

```diff
@@ -0,0 +1,13 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  Copyright (c) Vijay Tholpadi 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+
+public protocol TextureErrorHandler {
```

Comment:
This shouldn't be `public` as it isn't part of the public API 🙂

Reply:
Fixed 👍 😃